Library/Homebrew/cmd/install.rb

```diff
@@ -1,4 +1,4 @@
-#:  * `install` [`--debug`] [`--env=`(`std`|`super`)] [`--ignore-dependencies`|`--only-dependencies`] [`--cc=`<compiler>] [`--build-from-source`|`--force-bottle`] [`--devel`|`--HEAD`] [`--keep-tmp`] [`--build-bottle`] <formula>:
+#:  * `install` [`--debug`] [`--env=`(`std`|`super`)] [`--ignore-dependencies`|`--only-dependencies`] [`--cc=`<compiler>] [`--build-from-source`|`--force-bottle`] [`--devel`|`--HEAD`] [`--keep-tmp`] [`--build-bottle`] <formula> [`install-options`]:
```

Comment:
Should be `[<options> ...]` to indicate that these items are optional (square brackets), are a placeholder rather than a literal command (angle brackets), can be multiply specified (ellipsis), and hint at the use of `brew options <formulae>` to list which are available.

A suitable explanatory line might be ``Installation options specific to <formula> may be appended to the command, and can be listed with `brew options` <formula>.``